test/unit-tests/android-application-manager.ts

```diff
@@ -0,0 +1,63 @@
+import { AndroidApplicationManager } from "../../mobile/android/android-application-manager";
+import { Yok } from "../../yok";
+import { assert } from "chai";
+
+describe("android-application-manager", () => {
+
+	let testInjector: IInjector,
+		validTestInput: Array<string>,
+		expectedValidTestInput: Array<string>;
```

Comment:
There is no difference between `Array<string>` and `string[]`. I prefer to use `string[]` for consistency with the other code.